### ISSUE TITLE
sproutimport_seeds not found

### ISSUE THREAD

**Issue by Coysh** [barrelstrength/craft-sprout-import#18]
When going to the 'Weed' page - I get a Database Exception:

```
Database Exception – yii\db\Exception
SQLSTATE[42S02]: Base table or view not found: 1146 Table 'eauk-org.sproutimport_seeds' doesn't exist
The SQL being executed was: SELECT GROUP_CONCAT(id) ids, `type`, `details`, COUNT(1) as total, `dateCreated`
FROM `sproutimport_seeds`
GROUP BY `dateCreated`, `details`, `type`
ORDER BY `dateCreated` DESC
Error Info: Array
(
    [0] => 42S02
    [1] => 1146
    [2] => Table 'eauk-org.sproutimport_seeds' doesn't exist
)
↵
Caused by: PDOException
SQLSTATE[42S02]: Base table or view not found: 1146 Table 'eauk-org.sproutimport_seeds' doesn't exist
in /home/vagrant/sites/eauk-org/vendor/yiisoft/yii2/db/Command.php at line 1258
```

The table does exist, but is prefixed by craft_
![image](https://user-images.githubusercontent.com/5764356/38252590-7e5017f2-374c-11e8-8edc-238159a5743b.png)

**Comment by Coysh** [comment]
Unforutnatley I'm still getting the same issue:

![image](https://user-images.githubusercontent.com/5764356/38298867-76a34228-37f0-11e8-9c2a-58d6cf3bad15.png)

**Comment by BenParizek** [comment]
@Coysh This is resolved in v1.0.0-beta.6

Thanks for reporting.
